chainbase/src/main/java/org/tron/core/db/common/iterator/StoreIterator.java

```diff
@@ -93,14 +93,12 @@ public boolean valid() {
 
   @Override
   public byte[] getKey() {
-    checkState();
     checkValid();
     return dbIterator.peekNext().getKey();
   }
 
   @Override
   public byte[] getValue() {
-    checkState();
```

Comment:
There is a varible method `close` and a method with the same name, maybe you should use different names to distinguish them.

Reply:
No changes for now.